From bffdb29097c0ca14abf104c7c3f7b1a2828829e6 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 18 Aug 2014 12:19:52 +0100
Subject: [PATCH 01/10] add hostosinfo.h to installed headers

.. if you build Qt Creator with system qbs then this
header is needed by:
src/plugins/qbsprojectmanager/defaultpropertyprovider.cpp
---
 src/lib/corelib/tools/tools.pri | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib/corelib/tools/tools.pri b/src/lib/corelib/tools/tools.pri
index 3bf459d..aaebae2 100644
--- a/src/lib/corelib/tools/tools.pri
+++ b/src/lib/corelib/tools/tools.pri
@@ -87,6 +87,7 @@ qbs_enable_unit_tests {
         $$PWD/cleanoptions.h \
         $$PWD/codelocation.h \
         $$PWD/error.h \
+        $$PWD/hostosinfo.h \
         $$PWD/settings.h \
         $$PWD/settingsmodel.h \
         $$PWD/preferences.h \
-- 
2.3.5

